.git2gus/config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
@sejal-salesforce, I'd like to separate this config into a separate PR, so I start testing on it.